_posts/2021-01-08-2020-review.md

```diff
@@ -194,6 +194,7 @@ There is so much more to capture, and you really would have to read all the
 [release notes](/docs/current/release.html#releases-2020) in detail to know it
 all. To safe you from that, here are a few more noteworthy changes:
 
+* Support for materialized views
```

Comment:
This could be in SQL section. but @martint would know better

Reply:
Yeah .. I wasnt sure either so for now I added it that everything else section...

Reply:
Currently, they are very iceberg-specific, so I would phrase as:

"Experimental support for materialized views in Iceberg connector".

Reply:
Ok

Reply:
Done